packages/webapp/src/components/lists/ServiceList/columns.tsx

```diff
@@ -31,7 +31,14 @@ export function useColumns(onServiceClose: (service: Service) => void) {
 				name: t('serviceListRowActions.start'),
 				className: cx(styles.actionButton),
 				onActionClick(service: Service) {
-					navigate(history, ApplicationRoute.ServiceKiosk, { sid: service.id })
+					navigate(history, ApplicationRoute.ServiceEntry, { sid: service.id })
+				}
+			},
+			{
+				name: t('serviceListRowActions.startKiosk'),
+				className: cx(styles.actionButton),
```

Comment:
you don't really need the `cx()` here, but it's not an issue.

Reply:
I was copy pasting from the other buttons in this view, so I can probably cut cx entirely. Excellent catch.